api/osf_api.py

```diff
@@ -211,6 +211,21 @@ def upload_fake_file(session, node=None, name='osf selenium test file for testin
     return name, metadata
 
 
+def delete_addon_files(session, provider, current_browser, guid):
+    """Delete all files for the given addon.
+    """
+    files_url = '{}/v2/nodes/{}/files/{}/'.format(session.api_base_url, guid, provider)
+
+    data = session.get(url=files_url)
+
+    for file in data['data']:
+        if file['attributes']['kind'] == 'file':
+            delete_url = file['links']['delete']
+            file_name = file['attributes']['name']
+            if current_browser in file_name:
+                delete_file(session, delete_url)
```

Comment:
Making sure that I understand this correctly:

* We only delete files that are created by the current browser 👍 
* The file endpoint only lists files and folders in the OSF project root level, but this is totally fine as long as our selenium tests only create root-level files.
* The file endpoint has a default 10-item page count, this maybe a problem if each browser has more than 3 files (or less if there are folders in there). I would suggest increase the page size to fit the test need.

Reply:
Cool! We raised to 20 using a query parameter.